src/language/markdown-source-code.js

```diff
@@ -23,6 +30,104 @@ import { VisitNodeStep, TextSourceCodeBase } from "@eslint/plugin-kit";
 /** @typedef {import("@eslint/core").ParseResult<RootNode>} ParseResult */
 /** @typedef {import("@eslint/core").SourceLocation} SourceLocation */
 /** @typedef {import("@eslint/core").SourceRange} SourceRange */
+/** @typedef {import("@eslint/core").FileProblem} FileProblem */
+/** @typedef {import("@eslint/core").DirectiveType} DirectiveType */
+
+//-----------------------------------------------------------------------------
+// Helpers
+//-----------------------------------------------------------------------------
+
+const commentParser = new ConfigCommentParser();
+const configCommentStart =
+	/<!--\s*(eslint(?:-enable|-disable(?:(?:-next)?-line)?)?)/u;
```

Comment:
```suggestion
	/<!--\s*(eslint(?:-enable|-disable(?:(?:-next)?-line)?))/u;
```

As `eslint` config comments are not supported yet, it might be better to exclude them here. Otherwise, they would be reported when `noInlineConfig: true` is set although they are not inline config yet. Also, there should be a whitespace after `eslint` to be a config comment (e.g., `<!-- eslintfoo -->` is not a config comment) so the regex wouldn't be entirely correct even if it does target `eslint` comments too.